src/components/Articles/BlogCards.tsx

```diff
@@ -0,0 +1,126 @@
+import moment from 'moment';
+import Axios from '../../API/axios';
+import { useEffect, useState } from 'react';
+
+export interface articlesProp {
+  id?: number;
+  badge?: string;
+  imgUrl?: string;
+  title?: string;
+  writer?: string;
+  timestamp?: string;
+  videoUrl?: string;
+}
+
+const BlogCards: React.FC = () => {
+  const [articles, setArticles] = useState<articlesProp[]>([]);
+  // get request to get articles
+  const getArticles = async () => {
+    return await Axios.get('/articles');
+  };
+  // onmount hydrate articles state
+  useEffect(() => {
+    getArticles().then((response) => setArticles(response?.data));
```

Comment:
You can handle all fetching and parsing in the `getArticles` method and just get the data here

Reply:
Yea this is perfect